woocommerce/woocommerce
# Add needed packages to build e2e

### All Submissions:






### Changes proposed in this Pull Request:



Closes 7-gh-woocommerce/platform-private

There was an issue when building `e2e-utils` and `e2e-environment` where files where not being brought in.

### How to test the changes in this Pull Request:

1. Run `pnpm install` or `pnpm nx build @woocommerce/e2e-utils` and `pnpm nx build @woocommerce/e2e-environment`.
2. Ensure the files for the two packages are built correctly by checking `packages/js/e2e-environment/build/setup/jest.setup.js` and `packages/js/e2e-utils/build/index.js`. The files should have the imports.
3.

### Other information:




### Changelog entry

> Fix build process for e2e-environment and e2e-utils packages.

### FOR PR REVIEWER ONLY:
